src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectState.cs

```diff
@@ -171,12 +172,13 @@ RazorProjectEngine CreateProjectEngine()
             {
                 var configuration = HostProject.Configuration;
                 var rootDirectoryPath = Path.GetDirectoryName(HostProject.FilePath).AssumeNotNull();
-
+                var useRoslynTokenizer = LanguageServerFeatureOptions.UseRoslynTokenizer;
```

Comment:
nit: intentional blank line remove? seemed nice to separate var declaration/init block from the return, though I'd be fine either way.